### ISSUE TITLE
Hard coded SSH keys

### ISSUE THREAD

**Issue by stephenyeargin** [jameswhite/raspi#15]
@jameswhite's SSH keys are hard coded in the repository.

https://github.com/websages/raspi/blob/master/sd_build/post_dd_config#L68-L84

**Comment by jameswhite** [comment]
Yeah they are.

**Comment by stephenyeargin** [comment]
:trollface:

**Comment by jameswhite** [comment]
I'm :+1: If you want spike up a to remove them. They're just there for my convenience. We could pull from https://github.com/jameswhite.keys, and variable out the jameswhite or something.

**Comment by stephenyeargin** [comment]
Yeah, I'm compiling a list of things to configure and then we can come up with a way to store and load that in at run time.
